AstarteDeviceSDKCSharp/AstartePairingService.cs

```diff
@@ -42,14 +42,13 @@ public class AstartePairingService
         public AstartePairingService(string pairingUrl, string astarteRealm)
         {
             _astarteRealm = astarteRealm;
-            _pairingUrl = new Uri(pairingUrl);
+            _pairingUrl = new Uri($"{pairingUrl.TrimEnd('/')}/v1");
 
             if (_pairingUrl == null)
             {
                 throw new ArgumentNullException(nameof(_pairingUrl), "Pairing url is empty");
             }
 
```

Comment:
I am not sure about this check, the `Uri(String)` constructor already throws an `ArgumentNullException` if the `pairingUrl` is null or an `UriFormatException` if the `pairingUrl` is empty, so I think that this check can be avoided. What do you think?

Reply:
According to documentation, this null check can be removed.